RFC/src/RFC-0300_DAN.md

```diff
@@ -1 +1,205 @@
+"OPTIONAL" in this document are to be interpreted as described in [RFC 2119](http://tools.ietf.org/html/rfc2119).
+
+## Disclaimer
+
+The purpose of this document and its content is for information purposes only and may be subject to change or update
+without notice.
+
+This document may include preliminary concepts that may or may not be in the process of being developed by the Tari
+community. The release of this document is intended solely for review and discussion by the community regarding the
+technological merits of the potential system outlined herein.
+
+## Goals
+
+The goal of this RFC is to describe the key features of the Tari second layer, a.k.a the Digital Assets Network (DAN)
+
+## Related RFCs
+
+* [RFC-0100: Base layer](RFC-0100_BaseLayer.md)
+* [RFC-0310: Digital assets](RFC-0310_Assets.md)
+* [RFC-0311: Second layer consensus strategies](RFC-3011_SecondLayerConsensus.md)
+* [RFC-0312: Validator nodes](RFC-3012_ValidatorNodes.md)
+
+## Description
+
+### Abstract
+
+[Digital asset]s (DAs) are managed by committees of special nodes called [Validator node]s (VNs). VNs manage digital asset state change and ensure
+that the rules of the asset contracts are enforced.
+* VNs form a peer-to-peer communication network that together defines the Tari [Digital Asset Network] (DAN)
+* VNs register themselves on the base layer and commit collateral to prevent Sybil attacks.
+* Scalability is achieved by sacrificing decentralisation. Not *all* VNs manage *every* asset. Assets are managed by
+  subsets of the DAN, called VN committees. These committees reach consensus on DA state amongst themselves.
+* VNs earn fees for their efforts.
+* Digital asset contracts are not Turing complete, but are instantiated by [Asset Issuer]s (AIs) using Digital Asset templates that are defined
+  in the DAN protocol code.
+
+### Digital Assets
+
+* Digital asset contracts are *not* Turing complete, but are selected from a set of [DigitalAssetTemplate]s that govern
+  the behaviour of each contract type. e.g. there could be a Single-Use Token template for simple ticketing systems; a
+  Coupon template for loyalty programmes and so on.
+* The template system is intended to be highly flexible and additional templates can be added to the protocol periodically.
+* Asset issuers can register Top-Level Digital Issuer (TLDI) names on the base chain to help disambiguate similar
+  contracts and improve the signal-to-noise ratio from scam- or copy-cat contracts.
+
+An [Asset Issuer] (AI) will issue a Digital Assets by constructing a contract from one of the supported set of [DigitalAssetTemplate]s. The AI will choose
+ how large the committee of Validator Nodes will be for this DA and have the option to nominate [Trusted Node]s to be part of the VN committee for the DA.
+Any remaining spots on the committee will be filled by permissionless VNs that are selected according to a [CommitteeSelectionStrategy]. This ia a strategy
```

Comment:
This feels very dangerous to me, I am of the opinion that its either nominate all VN's or all are selected CommitteeSelectionStrategy. 

The problem of selecting some and not other is you open up the randomly selected node to be attacked. For example we have 2 nominated nodes and one randomly selected node. The nominated nodes are chosen bad actors. 

1. Selected nodes A and B which are bad actors make a bad unlawful change to the asset. 
2. Randomly selected node C, makes a good lawful change. 
3. In the committee Node C is the odd one out, A and B flag him as dishonest. 
4. A and B can then try to steal the collateral of C.

Reply:
This is where the Fraud Proof process could be used by the permissionless node. But yes perhaps its all permissioned  or all permissionless.